config/file.go

```diff
@@ -214,6 +214,7 @@ func (fs *FileStore) HasFile(name string) (bool, error) {
 func (fs *FileStore) RemoveFile(name string) error {
 	if filepath.IsAbs(name) {
 		// Don't delete absolute filenames, as may be mounted drive, etc.
+		mlog.Debug("Absolute filename will not be removed from the operationg system", mlog.String("filename", name))
```

Comment:
Typo with `operating`, but perhaps more clearly as:
```suggestion
		mlog.Debug("Skipping removal of configuration file with absolute path", mlog.String("filename", name))
```

Reply:
updated text